lib/archethic_web/explorer_router.ex

```diff
@@ -94,6 +99,11 @@ defmodule ArchethicWeb.ExplorerRouter do
     )
   end
 
+  scope "/api" do
```

Comment:
This scope should be before the other api scope, because the last `forward("/"...)` will catch all path after api including web_hosting. So `/api/web_hosting` is not accessible